### PR TITLE
[SPARK-516]Use bootstrap for spark local ip

### PR DESCRIPTION
**Pull request by ArtRand**

### CODE REVIEW

**Comment by ArtRand** [comment]
retest this please

File: universe/marathon.json.mustache [hunk 1]
```diff
@@ -7,7 +7,7 @@
 {{#security.kerberos.krb5conf}}
         "SPARK_MESOS_KRB5_CONF_BASE64": "{{security.kerberos.krb5conf}}",
 {{/security.kerberos.krb5conf}}
-
+        "NO_BOOTSTRAP": "true",
```
Comment (susanxhuynh) [inline universe/marathon.json.mustache]: If NO_BOOTSTRAP is set, then won't the `if [-z $NO_BOOTSTRAP]` above evaluate to false and skip the whole bootstrap lookup?

Comment (ArtRand) [reply universe/marathon.json.mustache]: that's what `if [ -n $NO_BOOTSTRAP]` would do `-z` returns `true` when the string length is zero or unset. I agree it's a double negative, but we don't want/need to run bootstrap for the dispatcher as there is a good amount of logic in `init.sh` and `runit/run` that takes care of it. ALSO, and perhaps more importantly, it's not broken there.

Comment (susanxhuynh) [reply universe/marathon.json.mustache]: Ahh, I get it now.

**Pull request merged**